js/service/shell/captureio.js

```diff
@@ -0,0 +1,53 @@
+// Copyright 2014-2015 runtime.js project authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+var events = require('events');
+exports.events = new events.EventEmitter();
```

Comment:
I'm not sure why captureio needed. It should be easier to just make custom object with properties
```
{
 write: function() {}
 readLine: function() {}
}
```
and pass that to command to capture io